File: bialydunajec-angular-frontend/projects/bialydunajec-main/src/app/academic-ministries-cottages/service/rest/dto/cottage-info.dto.ts
import {PlaceDto} from '../../../../shared/dto/place.dto';
import {CottageBossDto} from './cottage-edit.form-model';
import {AuditDto} from '../../../../shared/dto/audit.dto';

export class CottageInfoDto {
  cottageId: string;
  campRegistrationsEditionId: string;
  cottageType: string;
  academicMinistryId?: string;
  name: string;
  logoImageUrl?: string;
  buildingPhotoUrl?: string;
  place?: PlaceDto;
  cottageState: string;
  cottageBoss: CottageBossDto;
  audit: AuditDto;
}

File: backend/bialydunajec-registrations/dto/src/main/kotlin/org/bialydunajec/registrations/dto/CottageInfoDto.kt
package org.bialydunajec.registrations.dto

import org.bialydunajec.ddd.dto.sharedkernel.AuditDto
import org.bialydunajec.ddd.dto.sharedkernel.PlaceDto


data class CottageInfoDto(
        val cottageId: String,
        val campRegistrationsEditionId: String,
        val cottageType: String,
        val academicMinistryId: String?,
        val name: String,
        val logoImageUrl: String?,
        val buildingPhotoUrl: String?,
        val place: PlaceDto?,
        val cottageState: String,
        val cottageBoss: CottageBossDto?,
        val audit: AuditDto
) {
    companion object {
    }
}

